skbio/stats/distance/_base.py

```diff
@@ -366,6 +366,187 @@ def filter(self, ids, strict=True):
         filtered_data = self._data[idxs][:, idxs]
         return self.__class__(filtered_data, ids)
 
+    def _stable_order(self, ids):
+        """Obtain a stable ID order with respect to self
+
+        Parameters
+        ----------
+        ids : Iterable of ids
+            The IDs to establish a stable ordering for.
+
+        Returns
+        -------
+        np.array, dtype=int
+            The corresponding index values
+        """
+        # establish a stable ordering, and cache both the positions and label
+        # of each ID in the stable order.
+        id_order = sorted((self._id_index[i] for i in ids))
+        return np.array(id_order, dtype=int)
+
+    @experimental(as_of="0.5.4")
```

Comment:
Should this version be upped since the most recent release, which has already been released, is 0.5.5?

Reply:
Ah, good catch :)